src/store/modules/order/OrderState.ts

```diff
@@ -1,7 +1,8 @@
 export default interface OrderState {
   list: {
     orders: any[],
-    total: number
+    total: number,
+    items: number
```

Comment:
I think we can name it something like, total -> ordersCount, items -> itemsCount

Reply:
@adityasharma7 Sir WDYT?

Reply:
The noun does not have to be plural when naming variables. When naming a variable the noun should be universal, order is always "order", not "orders".

Reply:
Ok Sir

Reply:
Improved variable names.